amazon-kinesis-client/src/test/java/software/amazon/kinesis/lifecycle/ShutdownTaskTest.java

```diff
@@ -370,10 +316,24 @@ public final void testGetTaskType() {
         assertEquals(TaskType.SHUTDOWN, task.taskType());
     }
 
-    private List<ChildShard> constructChildShards() {
-        List<ChildShard> childShards = new ArrayList<>();
-        List<String> parentShards = new ArrayList<>();
-        parentShards.add(shardId);
+    private void verifyShutdownAndNoDrop() {
+        verify(recordsPublisher).shutdown();
+        verify(leaseCoordinator, never()).dropLease(any(Lease.class));
+        verify(shardRecordProcessor, never()).leaseLost(any(LeaseLostInput.class));
+    }
+
+    private Lease setupLease(final String leaseKey, final Collection<String> parentShardIds) throws Exception {
```

Comment:
Nit: While creating new methods, why not add a description to the method while its fresh?